src/test/rustdoc/const-display.rs

```diff
@@ -22,26 +30,36 @@ pub const fn foo2() -> u32 { 42 }
 #[rustc_const_stable(feature = "rust1", since = "1.0.0")]
 pub const fn bar2() -> u32 { 42 }
 
-// @has 'foo/fn.foo2_gated.html' '//pre' 'pub const unsafe fn foo2_gated() -> u32'
+
+// @has 'foo/fn.foo2_gated.html' '//pre' 'pub const fn foo2_gated() -> u32'
+// @!has - '//span[@class="since"]'
 #[unstable(feature = "foo2", issue = "none")]
-pub const unsafe fn foo2_gated() -> u32 { 42 }
+pub const fn foo2_gated() -> u32 { 42 }
```

Comment:
So, when you removed the duplicate test, you removed the most complicated one. So the end effect is that this tests less than before your PR.

Sorry to be so picky! This is the last round of review I think :)
```suggestion
pub const unsafe fn foo2_gated() -> u32 { 42 }
```